src/bp-blogs/classes/class-bp-blogs-template.php

```diff
@@ -163,8 +163,6 @@ public function __construct( ...$args ) {
 		// Set the total blog count.
 		if ( empty( $r['max'] ) || ( $r['max'] >= (int) $this->blogs['total'] ) ) {
 			$this->total_blog_count = (int) $this->blogs['total'];
-		} else {
-			$this->total_blog_count = (int) $max;
```

Comment:
Why do you remove the default `total_blog_count` value ?

Reply:
If you look at the code, you will see that the $max variable does not exist. This code is defaulting to 0 since `(int) null === 0`. Since my previous changes, the `$total_blog_count` property already defaults to 0, so this is not necessary.

Reply:
Source: https://github.com/buddypress/buddypress/blob/989ed99592aea76bafb0ffac9a5b32520603974c/src/bp-blogs/classes/class-bp-blogs-template.php#L180

Reply:
Then it's probably a bug that happened when one of us removed the use of the `extract()` function. If you look at this line https://github.com/buddypress/buddypress/blob/989ed99592aea76bafb0ffac9a5b32520603974c/src/bp-blogs/classes/class-bp-blogs-template.php#L130 It's possible to use a `max` argument in blogs queries, so `$max` should be `(int) wp_unslash( $r['max'] )`.

Reply:
👍🏾